nginx/Dockerfile

```diff
@@ -13,6 +13,7 @@ ENV NGINX_UPSTREAM_BLACKFIRE_PORT 9000
 ENV NGINX_ROOT                    /var/www/html
 ENV NGINX_PUBLIC                  ''
 ENV NGINX_TEMPLATE                application.conf
+ENV NGINX_RESOLVER                127.0.0.11
```

Comment:
This isn't my strong suit; what is 127.0.0.11?  Trying to determine if that should indeed be the default